features/recognize.js

```diff
@@ -172,7 +172,7 @@ async function handleGenericError(bot, message, error) {
 
 async function sendNotificationToReceivers(bot, message, gratitude) {
   for (let i = 0; i < gratitude.receivers.length; i++) {
-    await bot.startPrivateConversation(gratitude.giver.id);
+    await bot.startPrivateConversation(gratitude.receivers[i].id);
```

Comment:
Maybe I'm missing something, but this isn't removing a debug statement but just changing the recipient?

Reply:
Hmm, bad choice of branch name.

This statement is for debugging. It makes notifications intended for the recipients of fistbumps actually go to the giver of the fistbump. It's particularly useful for debugging locally so you can test that the notifications still work, without annoying anyone. I left that debugging change in the original PR though by mistake. 🤦‍♀️